src/parser/shared/modules/resources/resourcetracker/ResourceTracker.ts

```diff
@@ -180,7 +180,7 @@ class ResourceTracker extends Analyzer {
     return this.getResource(event)?.cost;
   }
 
-  getResource(event: CastEvent | HealEvent | EnergizeEvent ) {
+  getResource(event: CastEvent | HealEvent | EnergizeEvent | DamageEvent) {
```

Comment:
Was this extra type actually used in the end anywhere? I thought they were both CastEvents that had the actual classResource props

Reply:
Oh good catch, I did at one point need to call getResource on my damage event but took it out a few iterations later and forgot to change that back